redbeams/src/main/java/com/sequenceiq/redbeams/service/network/NetworkParameterAdder.java

```diff
@@ -74,4 +94,12 @@
         }
         return parameters;
     }
+
+    private String getAzureSubnetToUseWithPrivateEndpoint(DetailedEnvironmentResponse detailedEnvironmentResponse, DBStack dbStack) {
+        String subscriptionId = subnetListerService.getAzureSubscriptionId(detailedEnvironmentResponse.getCrn());
+        return subnetChooserService.chooseSubnetForPrivateEndpoint(detailedEnvironmentResponse.getNetwork().getSubnetMetas().values(), dbStack).stream()
+                .findFirst()
+                .map(csn -> SubnetListerService.expandAzureResourceId(csn, detailedEnvironmentResponse, subscriptionId))
```

Comment:
`SubnetListerService#expandAzureResourceId` should not be static, and it should be public and used via spring injection.

Reply:
Ok, I happily de-staticise.

Reply:
thanks